openmdao/utils/units.py

```diff
@@ -845,6 +845,11 @@ def _update_library(cfg):
 _UNIT_CACHE = {}
 
 
+def _is_unitless(units):
+    unit_meta = _find_unit(units)
```

Comment:
_find_unit can return None if units are not valid, so maybe just to be safe, `return unit_meta is not None and unit_meta.is_dimensionless()`